CMakeLists.txt

```diff
@@ -461,6 +461,10 @@ if (FLB_AWS)
   FLB_DEFINITION(FLB_HAVE_AWS)
 endif()
 
+if (FLB_AWS_ERROR_REPORTER)
+  FLB_DEFINITION(FLB_HAVE_AWS_ERROR_REPORTER)
+endif()
+
```

Comment:
I think you need to also declare the flag as an option and make it default to off, like as shown here: https://github.com/fluent/fluent-bit/blob/master/CMakeLists.txt#L78